assets/startup.sh

```diff
@@ -38,29 +38,30 @@ if [ -z "${LOG_LEVEL}" ]; then
   LOG_LEVEL=0
 else
   # confirm log level is between 0-3, since these are the only log levels supported
-  if [ "${LOG_LEVEL}" -gt 3 ]; then
+  if [[ $LOG_LEVEL != [0123] ]]; then
```

Comment:
While the double-square-brackets are a bashism, the stricter check is an improvement, so how about using the POSIX-compliant `expr` to do that? (tested to work under busybox-sh)
```suggestion
  if expr "${LOG_LEVEL}" : [^0123] > /dev/null ; then
```
I admit that this is uglier.